gate-web/src/main/groovy/com/netflix/spinnaker/gate/security/rolesprovider/UserRolesProvider.groovy

```diff
@@ -18,4 +18,6 @@ package com.netflix.spinnaker.gate.security.rolesprovider
 
 public interface UserRolesProvider {
   public Collection<String> loadRoles(String userEmail)
+
+  public Map<String, Collection<String>> multiLoadRoles(Collection<String> userEmails)
```

Comment:
Added javadoc. Making a first class `UserRoles` make the mapping to `User` harder so I left it out.